packages/docz-tools/src/gatsby-theme-docz/components/Sidebar/index.tsx

```diff
@@ -11,13 +11,13 @@ import { DeferRender } from "../DeferRender";
 export function Sidebar() {
   const [query, setQuery] = useState("");
   const {
-    themeConfig: { sideBarWidth },
+    themeConfig: { sideBarWidth, hideLogo, isFixedHeight },
```

Comment:
perhaps `hasLogo` and `sidebarOffset`?

Reply:
- `hideLogo` ->`hasLogo`, set to `true` in theme configuration
- `isFixedHeight` ->`sidebarOffset`, defaulted to `0`